### DISCUSSION TITLE
Pheonix Live Preview not working, Can't open linux electron app

### DISCUSSION THREAD

**Discussion by JessMTermini** [phcode-dev/phoenix#1105] [Q&A]
Live preview is not working for Phoenix. It only shows changes after both the edited file is manually saved AND then there is a manual browser page reload...why is this happening? Is this happening only for me? This was the main feature that brought me to Brackets in the 1st place. Without this working it almost makes it pointless. Oh and also the electron Brackets app won't open after installation on my Chrome OS Linux container running Debian 11 Bullseye.

**Comment by abose** [comment]
Hi @JessMTermini what browser and os is used?

Close all tabs with phcode and Reloading phcode.dev two times will likely fix the live preview issue if any in chrome/edge/opera.

Phoenix doesn't currently ship with any electron app, can you give details on where you got the electron app?
